frontend/templates/views/partials/level-toggle.j2

```diff
@@ -0,0 +1,55 @@
+{% do doc.styles.addCssFile('css/components/molecules/level-toggle.css') %}
+{% set level = doc.level %}
+{% set url = doc.url.path %}
+
+
+
+<amp-state id="filter" class="" hidden="" aria-hidden="true">
+  <script type="application/json">
+  {
+    "level": "[= level =]"
+  }
+  </script>
+</amp-state>
+
+
+<div class="ap-m-level-toggle">
+  <div class="ap-m-level-toggle-elements">
+    <div class="toggle-button">
+      <input
+      [% if level == 'advanced' %]
```

Comment:
nit: I'd prefer to only switch on the content that changes and avoid duplicating strings